packages/create-app/README.md

```diff
@@ -0,0 +1,23 @@
+# @backstage/create-app
+
+This package provides a CLI for creating apps.
+You can use the flag `--skip-install` to skip the install.
+
+## Installation
+
+Install the package via npm or yarn:
+
+```sh
+$ npm install --save @backstage/create-app
```

Comment:
Let's switch this to instructions for how to use it via `npx` instead, since that's how it's going to be consumed pretty much always nowadays :grin:. The alternative would be to install it globally and use it directly.

So some working around using `npx @backstage/create-app` :>